packages/xo-server-test-plugin/index.js

```diff
@@ -42,8 +42,9 @@ exports.default = function (opts) {
     //
     // Note: before being called, the configuration is validated
     // against the provided configuration schema.
-    configure: function (configuration) {
+    configure: function (configuration, loaded) {
```

Comment:
Should probably be an object containing a `loaded` property instead of directly a `boolean`.
It will allow us to pass more values in the future if necessary.